src/workloads/execution/locf.yml

```diff
@@ -0,0 +1,257 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests the performance of $locf. The benchmark operations test numeric,
+  string, date and randomized data types. There is a test without partitions, with a 
+  single partition, and with multiple partitions for each data type.
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 10000
+    BatchSize: &batchSize 100
+    Document:
+      part1: {^Choose: {from: [1, 2, 3, 4]}}
+      part2: {^Choose: {from: [1, 2, 3, 4]}}
+      numeric: {^Choose: {from: [
+        {^RandomInt: {min: -100, max: 100}}, 
+        {^RandomDouble: { min: 0.0, max: 500.0 }},
+        null], weights: [1, 1, 8]}}
```

Comment:
Any particular reason for these weights? Not saying it should be changed, just curious.

Reply:
I think when I was discussing nulling documents with David Percy we mentioned making it 70% likely for a value to be null, so I was just trying to keep near that likelihood?